components/src/components/Avatar/Avatar.docs.mdx

```diff
@@ -51,6 +51,17 @@ import { Avatar } from 'degen'
 </Stack>
 ```
 
+## Border
+
+Display a light border around the edge to define image edges on similarly colored backgrounds. Defaults to `true`.
+
+```tsx live=true
+<Stack direction="horizontal">
+  <Avatar src={avatars.noun97} />
+  <Avatar border={false} src={avatars.noun97} />
```

Comment:
If `border` is default `true`, might be nice to change this prop name to `noBorder` instead of `border={false}`.